stylesheet/index.css

```diff
@@ -0,0 +1,287 @@
+@import url(main.css);
+
+.main--banner-container {
+    display: flex;
+
+    flex-direction: column;
+
+    position: relative;
+
+    height: auto;
+    max-width: 100%;
+
+    padding-bottom: 10%;
+}
+
+.main--quote-container{
+    display: flex;
+
+    flex-direction: column;
+
+    text-align: center;
+
+    color: var(--color-text);
+
+    position: relative;
+
+    margin-top: 120px;
+} 
+
+.main--quote {
+    font-size: 40px;
+
+    width: 50%;
+
+    margin: auto;
+
+    position: relative;
+}
+
+.main--quote-bold {
+    font-weight: bolder;
+}
+
+.main--banner-img{
+
+    max-width: 100%;
+
+    height: auto;
+
+    position: absolute;
+
+    padding: 0;
+}
+
+.button {    
+    width: 200px;
+    height: 40px;
+
+    margin: auto;
+
+    color: var(--color-text);
+
+    border: 0px;
+
+    background-image: linear-gradient(to right, red , orange);
+
+    cursor: pointer;
+}
+
+.main--subHeading {
+    padding: 0;
+    margin: 20px 0 30px 0;
+}
+
+.main--info-container {
+    display: flex;
+
+    flex-direction: column;
+    
+    text-align: center;
+
+    color: var(--color-text);
+
+    position: relative;
+}
+
+.main--info-paragraph {
+    font-size: 15px;
+
+    line-height:1.6;
+
+    width: 45%;
+
+    margin: 30px auto;
+
+    z-index: 2;    
+}
+
+.main--info-img {
+    position: absolute;
+
+    max-width: 100%;
+    height: auto;
+
+}
+
+.main--flex-containers {
+    display: grid;
+    grid-template-columns: auto;    
+
+}
+
+.main--info-heading {
+    z-index: 2;
+
+    margin-top: 90px;
+}
+
+.main--flex-images-container{
+    display: flex; 
+    flex-direction: row;
+
+    position: relative;
+
+    margin: 80px auto auto auto;
+
+    color: var(--color-text);
+}
+
+.main--flex-img {
+    width: 100%;
+    height: 100%;
+
+    position: relative;
+}
+
+.main--flex-div {
+    height: 100%;
+    width: auto;
+
+    display: flex;
+    flex-direction: column;
+}
+
+.main--boatImg-text{
+    position: absolute;
+}
+
+.main--vaultariBackground-img{
+    width: 100%;
+    height: 100%;    
+}
+
+.main--description-img {
+    width: 100%;
+    height: 100%;    
+}
+
+.main--feedback-container {
+    margin: 0;
+
+    position: relative;
+}
+
+.main--description-container {
+    position: relative;
+    
+    margin: 0;
+}
+
+.main--laptop-img {
+    width: 100%;
+    height: 100%;
+}
+
+.main--partnersBackground-img {
+    width: 100%;
+    height: 100%;
+}
+
+.main--contact-container{
+    background-color: #000c2f;
+}
+
+.main--contact-images-container {
+    display: grid;
+    grid-template-areas: 
+    "logo logo logo"
+    "social1 social2 social3"; 
```

Comment:
habar n am ce face asta .. deci deja esti mai avansat ca mine 💪 ... da o sa caut sa aflu

Reply:
am pus logo-urile in forma aia, gen logo-ul companiei, si jos dupa cum vezi, fb, linked si twitter, cred